homeassistant/components/waqi/air_quality.py

```diff
@@ -114,6 +117,7 @@ def device_state_attributes(self):
 
         return data
 
+    @Throttle(SCAN_INTERVAL)
```

Comment:
Why move the throttle to the entity? Shouldn't we throttle all entities in the same time like before by throttling in the component?